### PR TITLE
fix: Updating docs

### PR DESCRIPTION
**Pull request by AlseinX**
Updating documents to introduce dependencies and features more properly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,57 +23,86 @@ An enterprise-class UI components based on Ant Design and Blazor.
...
+
```
Comment (ElderJames) [inline README.md:83]: You don't have to write it twice.

File: README.md [hunk 2]
```diff
@@ -149,6 +165,19 @@ Regularly synchronize with Official Ant Design specifications, you can check the
...
+- Install Node.js (This is needed for transpiling style sheets and interoperable TypeScript codes)
+- Clone the repository
+
+  ```bash
+  $ git clone git@github.com:ant-design-blazor/ant-design-blazor.git
+  $ cd ant-design-blazor
+  $ npm install
+  $ npm start
+  ```
+
```
[the remark this answers is not in the document]
Comment (AlseinX) [reply README.md:180]: Just for respectively specify the difference of the dependencies. I do think there a must to write then separately.

[the remark this answers is not in the document]
Comment (AlseinX) [reply README.md:180]: If every single item should be conditional, I think it surely better to group by the hosting model.

**Comment by ElderJames** [comment]
I think it would be better to have it in the [documentation](https://github.com/ant-design-blazor/ant-design-blazor/tree/master/docs).

**Pull request closed without merging**